.gitignore

```diff
@@ -62,7 +62,7 @@ target/
 
 # Virtualenv
 .venv
-.pytest-pylint
+.pytest_pylint
```

Comment:
I don't think this repo adds this folder. If it's something local to you, could you add it to your global ~/.gitignore instead?

Reply:
Sure. Done

Reply:
I added because I used this folder as a suggestion for virtualenv name in the docs